cmd/database.go

```diff
@@ -55,12 +59,21 @@ func getPsqlConnInfo() (PsqlConnInfo, error) {
 		return PsqlConnInfo{}, fmt.Errorf("%s_PGSQL_PASSWORD env var is required", envVarsPrefix)
 	}
 
+	var schema string
+	if value := viper.GetString("PGSQL_SCHEMA"); value != "" {
+		schema = value
+	} else {
+		schema = ""
+		fmt.Printf("No schema set.")
```

Comment:
Use log instead of print:

```suggestion
		glog.V(2).Info("No schema set.")
```

Reply:
Thanks for the suggestion @mozillazg.